packages/lambdas/src/cda-to-visualization.ts

```diff
@@ -37,12 +37,18 @@ export const handler = Sentry.AWSLambda.wrapHandler(
     console.log(`Running with conversionType: ${conversionType}, fileName: ${fileName}`);
 
     const originalDocument = await downloadDocumentFromS3({ fileName, bucketName });
-    if (!originalDocument) {
+    if (originalDocument === undefined) {
       throw new MetriportError(`Document not found in S3`, undefined, {
```

Comment:
We should throw a NotFoundError here, eh? Just  need to test the upstream effects.